test/sql.test.js

```diff
@@ -55,6 +59,17 @@ describe('sql connector', function() {
     expect(column).to.eql('NAME');
   });
 
+  it('should map column name from name attribute', function() {
+    var column = connector.column('customer', 'primaryAddress');
+    expect(column).to.eql('primary_address');
+  });
+
+  it(`should not map column name from name attribute 
+    if db specific column name provided`, function() {
```

Comment:
We don't use multi-line test names. If the test name cannot fit into a single line, then use string concatenation please.

Personally, I'd reword the test name as follows:

```js
it('prefers database-specific column name over property name', function() {
  // ...
});
```